Sources/tart/OCI/Manifest.swift

```diff
@@ -13,6 +13,7 @@ let nvramMediaType = "application/vnd.cirruslabs.tart.nvram.v1"
 // Manifest annotations
 let uncompressedDiskSizeAnnotation = "org.cirruslabs.tart.uncompressed-disk-size"
 let uploadTimeAnnotation = "org.cirruslabs.tart.upload-time"
+let diskFormatLabelAnnotation = "org.cirruslabs.tart.disk.format"
```

Comment:
Not: this is a label, not an annotation.

So it should probably go to `// Manifest labels` section and be named `diskFormatLabel`.